### PR TITLE
Lighter iotagent-json implementation

### PR DESCRIPTION
**Pull request by giovannicuriel**
This new code is intended to be simpler (and hopefully faster) than the original one. It doesn't require any extra configuration (such as creating devices) - for each new message that it receives, it asks (or, at least, will ask) DeviceManager if that device exists and which is its ID (and related fiware-service, for now). After receiving a valid answer (the device indeed exists), it sends an update message to Orion, just like before (almost - it now uses the simpler Orion's v2 API).

Also, the code is now in TypeScript. Any developer with some knowledge with JavaScript will have no problem reading it.

This PR is part of dojot/dojot#95

### CODE REVIEW

File: src/device-manager-handler.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import fs = require("fs");
```
Comment (mmagr) [inline src/device-manager-handler.ts]: this isn't used

**Review by mmagr — DISMISSED** [review]
Looks nice, and almost 10x smaller!

File: src/device-manager-handler.ts (hunk 1 above)
Comment (giovannicuriel) [reply src/device-manager-handler.ts]: Ok!

**Review by mmagr — DISMISSED** [review]

**Review by mmagr — APPROVED** [review]
... and now travis checks stuff :relieved:

**Pull request merged**